src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -124,6 +128,10 @@ public GuiManager(Main main, PluginAPI pluginAPI, RepairManager repairManager) {
         this.chat = register("chat", ChatGui.class);
         this.settingsGui = register("settings", SettingsGui.class);
 
+        this.dispatch = register("dispatch");
+        this.icon = register("popup_generic_icon");
+        this.iconOk = register("popup_generic_icon_ok");
```

Comment:
these should probs be using custom gui sub-classes, which provide some extra methods that allow some level of interaction with the gui